validators/fuji/fuji-versions.md

```diff
@@ -88,7 +88,7 @@ MONITORING_CONTAINERS=True
 DOCKER_LVMPY_STREAM=1.0.1-beta.1
 MANAGER_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/rinkeby/skale-manager/1.8.0/skale-manager-1.8.0-beta.1-rinkeby-abi.json
 IMA_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/rinkeby/IMA/1.0.0-beta.2/abi.json
-CONTAINER_CONFIGS_STREAM=1.6.3-testnet
+CONTAINER_CONFIGS_STREAM=1.6.4-testnet
```

Comment:
We need also update version in validators/fuji/soft-upgrade-testnet.md (142 line)